### DISCUSSION TITLE
Currently showing two circles above

### DISCUSSION THREAD

**Discussion by tenSunFree** [rubenquadros/Jetpack-Compose-Video-Games-Example#9] [General]
Currently showing two circles above.
![Screenshot_20220116-234700](https://user-images.githubusercontent.com/23090085/149667456-e759b8f6-a1db-4e7f-94eb-707d92b6e914.png)

Have you considered adding the relevant judgment of when, so that only one circle is displayed, and it is displayed in the middle?
<img width="490" alt="2022-01-16_235333" src="https://user-images.githubusercontent.com/23090085/149667476-bb610fb6-cd80-4193-ad5d-a28ddac8eec4.png">
![Screenshot_20220116-234937](https://user-images.githubusercontent.com/23090085/149667482-d42db3b5-311f-4593-8b68-1bda726c4555.png)

**Comment by rubenquadros** [comment]
The 2 progress bars are by design. Will consider changing it to a single progress bar in center for first time launch
